examples/workbooks/LeadsExample.ts

```diff
@@ -0,0 +1,324 @@
+import * as FF from "@flatfile/configure";
```

Comment:
for each of these examples I'd start with a comment block that describes the example more holistically